debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9ObjectHelper.java

```diff
@@ -69,26 +74,47 @@ public static U32 flags(J9ObjectPointer objPointer) throws CorruptDataException
 	{		
 		if(J9BuildFlags.interp_flagsInClassSlot) {
 			long bitmask = (J9Consts.J9_REQUIRED_CLASS_ALIGNMENT - 1);
-			J9ClassPointer clazz = objPointer.clazz();
+			UDATA clazz = rawClazz(objPointer);
 			
-			return new U32(UDATA.cast(clazz).bitAnd(bitmask));
+			return new U32(clazz.bitAnd(bitmask));
 		} else {
 			throw new UnsupportedOperationException("Only builds with interp_flagsInClassSlot currently supported.");
 		}
 	}
 
+	public static long headerSize()
+	{
+		if (mixedModeSupport) {
+			if (J9JavaVMHelper.compressObjectReferences) {
+				return J9ObjectCompressed.SIZEOF;
+			}
+			return J9ObjectFull.SIZEOF;
+		}
+		return J9Object.SIZEOF;
+	}
+
+	public static UDATA rawClazz(J9ObjectPointer objPointer) throws CorruptDataException
+	{
+		UDATA clazz = null;
```

Comment:
Initialization here has no benefit; why not separate `return` statements like above?

Reply:
Done.